dashboard/modules/job/job_manager.py

```diff
@@ -67,7 +83,10 @@ def __init__(self, job_id: str, metadata: Dict[str, str]):
         self._runtime_env = ray.get_runtime_context().runtime_env
 
         self._metadata = metadata
-        self._metadata[JOB_ID_METADATA_KEY] = job_id
+        if JOB_ID_METADATA_KEY not in self._metadata:
+            self._metadata[JOB_ID_METADATA_KEY] = job_id
+        if JOB_NAME_METADATA_KEY not in self._metadata:
+            self._metadata[JOB_NAME_METADATA_KEY] = job_id
```

Comment:
how we're planning to use this field if we assign job_id to it, same as `JOB_ID_METADATA_KEY` ?